solution/ui/e2e/cypress/e2e/search.spec.cy.js

```diff
@@ -84,81 +89,12 @@ describe("Search flow", () => {
-            fixture: "meta-data.json"
-        }).as("metadata");
-        cy.viewport("macbook-15");
-        cy.visit(`/search/?q=${SEARCH_TERM2}`, { timeout: 60000 });
-        cy.wait("@resources");
-
-        cy.get(".reg-results-content .search-results-count > h2").should(
-            "have.text",
-            "Regulations"
-        );
-        cy.get(".reg-results-content .search-results-count > span").should(
-            "be.visible"
-        );
-        cy.get(".resources-results-content .search-results-count > h2").should(
-            "have.text",
-            "Resources"
-        );
-        cy.get(".resources-results-content .search-results-count > span > span").should(
-            "have.text",
-            " 1 - 1 of "
-        );
-        cy.get(
-            ".resources-results-content .search-results-count > span"
-        ).should("be.visible");
-        cy.get(
-            ".reg-results-content .reg-results-container .result:nth-child(1) .result__link"
-        ).should("be.visible");
-        cy.get(
-            ".reg-results-content .reg-results-container .result:nth-child(1) .result__link a"
-        ).should("have.attr", "href");
-    });
-
-    it("should return no resources when going on page 3 without enough resources returned", () => {
-        cy.intercept(`**/v3/resources/search?q=${SEARCH_TERM2}**page=3**`, {
-            fixture: "search-gov-not-enough.json",
-            statusCode: 400,
-        }).as("resources");
-        // this call is never executed since the conditional in js will not call it.
-        cy.intercept(`**/v3/resources/?&**${SEARCH_TERM2}**page=3**`, {
-            fixture: "meta-data.json"
-        }).as("metadata");
-        cy.viewport("macbook-15");
-        cy.visit(`/search/?q=${SEARCH_TERM2}&page=3`, { timeout: 60000 });
-        cy.wait("@resources");
-
-        cy.get(".reg-results-content .search-results-count > h2").should(
-            "have.text",
-            "Regulations"
-        );
-        cy.get(".reg-results-content .search-results-count > span").should(
-            "be.visible"
-        );
-        cy.get(".resources-results-content .search-results-count > h2").should(
-            "have.text",
-            "Resources"
-        );
-        cy.get(".resources-results-content .search-results-count > span").should(
-            "have.text",
-            " 0 results"
-        );
-        cy.get(
-            ".resources-results-content .search-results-count > span"
-        ).should("be.visible");
-        cy.get(
-            ".reg-results-content .reg-results-container .result:nth-child(1) .result__link"
-        ).should("be.visible");
-        cy.get(
-            ".reg-results-content .reg-results-container .result:nth-child(1) .result__link a"
-        ).should("have.attr", "href");
+    it("should have the correct labels for public and internal documents", () => {
+        cy.checkPolicyDocs({
+            username,
```

Comment:
Since the Resources column (the right column) on the search page now requests and displays data the exact same way as the Subjects page, I've moved the assertions for those results to a utility method that can be called in both the Search and Subjects test suites.